bin/dev-setup.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+'use strict';
+
```

Comment:
Although I saw some other build scripts had eslint ignored, I thought it was actually useful to keep on here -- it helped me better conform to the project style.